app/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useRef} from 'react';
+import ArticleDetails from 'components/ArticleDetails';
+import ArticleList from 'components/ArticleList';
+import {useNavigate, useParams} from 'react-router-dom';
+
+import styles from './Home.module.scss';
+import {useArticlesContext} from 'context/articles/ArticleContext';
+import {ArticlesContextInterface} from 'context/articles/typings';
+
+function Home() {
+  const {page} = useParams();
+  const navigate = useNavigate();
+  const {articles, pagination} =
+    useArticlesContext() as ArticlesContextInterface;
```

Comment:
To consider why we need to impose a type here and use 'as ArticlesContextInterface'.